### PR TITLE
#91200 [Admin] [User] List user

### PR DESCRIPTION
**Pull request by ngovanhieu**
Ảnh review giao diện ạ
http://www.upsieutoc.com/image/RMna0q

### CODE REVIEW

**Comment by DungNQ-Framgia** [comment]
OK

File: resources/views/admin/users/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    <td>
+                    @if ($user->isMember())
+                        {!! Form::open(['action' => ['Admin\UsersController@destroy', $user->id],
+                            'method' =>  'DELETE']) !!}
+                        <div class="btn-group-sm">
+                            <a href="{{ action('Admin\UsersController@edit', ['id' => $user->id]) }}" class="btn btn-warning">
+                                {{ trans('admin/users/index.buttons.edit') }}
+                            </a>
+                            {!! Form::submit(trans('admin/users/index.buttons.delete'), 
+                                ['class' => 'btn btn-danger']) !!}
```
Comment (dung13890) [inline resources/views/admin/users/index.blade.php]: style code convention

**Comment by dung13890** [comment]
ok

**Comment by hungcoj** [comment]
Ok.

**Pull request merged**